terraform/environment/dynamodb.tf

```diff
@@ -9,13 +9,13 @@ data "aws_kms_alias" "dynamodb_encryption_key_eu_west_2" {
 }
 
 resource "aws_dynamodb_table" "lpas_table" {
-  name         = "${local.environment_name}-Lpas"
+  name         = "${local.environment_name}-Lpas2"
```

Comment:
need this to be recreated, and the actual name doesn't really matter so... added a 2 😄

Reply:
Is the issue here that terraform won't recreate the resource when changing the hash key?

Reply:
yeah you can't change the keys on an existing table

Reply:
That would have resulted in a recreation anyway